.env

```diff
@@ -0,0 +1 @@
+DB_CONN="mongodb://localhost:27017/cyberpizzadb"
```

Comment:
@IgorChaikin `.env` file should not be stored in git, you can store `.env.sample` with examples of environment variables